sky/skylet/log_lib.py

```diff
@@ -403,7 +404,9 @@ def tail_logs(job_owner: str,
         time.sleep(_SKY_LOG_WAITING_GAP_SECONDS)
         status = job_lib.update_job_status(job_owner, [job_id], silent=True)[0]
 
-    if status in [job_lib.JobStatus.RUNNING, job_lib.JobStatus.PENDING]:
+    if follow and status in [
```

Comment:
Maybe worth to document this arg in this func's docstr. Not entirely sure my other comment is correct.

Reply:
Added. Thanks!